using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -103,116 +119,150 @@ This API synchronizes the provided customer identifier type key and value with t
+#### Syntax
 
 ```jsx
-ACPIdentity.syncIdentifier(identifierType, identifier, authenticationState);
+ACPIdentity.syncIdentifier(identifierType: String, identifier: String, authenticationState: string)
 ```
 
-### syncIdentifiers
+o *identifierType* (String) containing identifier type, and the *identifier* (String) containing identifier value; should not be null or empty. If the identifierTtype and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+o  *authenticationState* (VisitorIDAuthenticationState) value indicating authentication state for the user contaisn one of the VisitorID.AuthenticationState values `ACPMobileVisitorAuthenticationState.AUTHENTICATED`,`ACPMobileVisitorAuthenticationState.LOGGED_OUT` and `ACPMobileVisitorAuthenticationState.UNKNOWN`
+
+#### Example
 
 ```jsx
-ACPIdentity.syncIdentifiers({"id1": "identifier1"});
+ACPIdentity.syncIdentifier(identifierType, identifier, ACPMobileVisitorAuthenticationState.AUTHENTICATED);
 ```
 
-### syncIdentifiersWithAuthState
+### syncIdentifiers<a id="syncIdentifiers-js"></a>
+
+The function of this `syncIdentifiers()`  API is same as the `syncIdentifier` API, which passes a list of identifiers as a dictionary each containing an identifier type as the key and an identifier as the value. In each of the identifiers pair, both identifier type and identifier should be non empty and non null values, otherwise they will be ignore. 
+
+#### **Syntax**
 
 ```jsx
-import {ACPMobileVisitorAuthenticationState} from '@adobe/react-native-acpcore';
+ACPIdentity.syncIdentifiers: (nullable NSDictionary*) identifiers;
+```
 
-ACPIdentity.syncIdentifiersWithAuthState({"id1": "identifier1"}, ACPMobileVisitorAuthenticationState.UNKNOWN);
+o   *identifiers* is a dictionary that contains Identifiers with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+#### Example**
+
+```jsx
+ACPIdentity.syncIdentifiers({"id1": "identifier1"});
+```
+
+### syncIdentifiersWithAuthState<a id="syncIdentifiersWithAuthState-js"></a>
+
+The function of this `syncIdentifiersWithAuthState()`  API is same as the `syncIdentifier` API, which  passes a list of identifiers as a dictionary and  the authenticationState. The identifiers dictionary contains identifiers; each containing an identifier type as the key and an identifier as the value. In each of the identifiers pair, both identifier type and identifier should be non empty and non null values, otherwise they will be ignore. 
```

Comment:
- Please change the beginning of the 1st sentence to "The function of this API is..."
- Delete the rest of the content and make `syncIdentifier` a hyperlink that points to the `syncIdentifier` section.